x-pack/plugins/security_solution/server/lib/detection_engine/signals/__mocks__/es_results.ts

```diff
@@ -109,6 +109,24 @@ export const sampleDocNoSortId = (
   sort: [],
 });
 
+export const sampleDocSeverity = (
+  severity?: Array<string | number | null> | string | number | null
+): SignalSourceHit => ({
+  _index: 'myFakeSignalIndex',
+  _type: 'doc',
+  _score: 100,
+  _version: 1,
+  _id: sampleIdGuid,
+  _source: {
+    someKey: 'someValue',
+    '@timestamp': '2020-04-20T21:27:45+0000',
```

Comment:
That doesn't look like a Zulu? It's fine if you want to test other values, just mentioning

Reply:
I had used a similar mock from above, that's all.